src/snyk/common/languageServer/settings.ts

```diff
@@ -0,0 +1,59 @@
+export type InitializationOptions = ServerSettings & {
```

Comment:
Yeah, this looks like it's meant to be used in the actual LanguageServer class ;)

Reply:
You mean having similar type declaration?

Reply:
Yeah